index.js

```diff
@@ -2,14 +2,21 @@
 'use strict';
 
 var path = require('path');
+var Funnel = require('broccoli-funnel');
+var MergeTrees = require('broccoli-merge-trees');
 
 module.exports = {
   name: 'ember-cli-js-cookie',
 
   included: function(app) {
     this._super.included.apply(this, arguments);
+    this.import('vendor/js.cookie.js');
```

Comment:
No need. I'll bump a major version in npm to reduce impact. Dependency issues are no fun 🎉